docs/use-cases/using-Intel-QAT-and-kata.md

```diff
@@ -3,54 +3,53 @@
+        * [Expose and Bind Intel® QAT virtual functions to VFIO-PCI (Every reboot)](#expose-and-bind-intel-qat-virtual-functions-to-vfio-pci-every-reboot)
+        * [Check Intel® QAT virtual functions are enabled](#check-intel-qat-virtual-functions-are-enabled)
     * [Prepare Kata Containers](#prepare-kata-containers)
         * [Download Kata kernel Source](#download-kata-kernel-source)
         * [Build Kata kernel](#build-kata-kernel)
         * [Copy Kata kernel](#copy-kata-kernel)
         * [Prepare Kata root filesystem](#prepare-kata-root-filesystem)
-        * [Compile Intel QAT drivers for Kata Containers kernel and add to Kata Containers rootfs](#compile-intel-qat-drivers-for-kata-containers-kernel-and-add-to-kata-containers-rootfs)
+        * [Compile Intel® QAT drivers for Kata Containers kernel and add to Kata Containers rootfs](#compile-intel-qat-drivers-for-kata-containers-kernel-and-add-to-kata-containers-rootfs)
         * [Copy Kata rootfs](#copy-kata-rootfs)
-        * [Update Kata configuration to point to custom kernel and rootfs](#update-kata-configuration-to-point-to-custom-kernel-and-rootfs)
-    * [Verify Intel QAT works in a Docker Kata Containers container](#verify-intel-qat-works-in-a-docker-kata-containers-container)
-    * [Build OpenSSL Intel QAT engine container](#build-openssl-intel-qat-engine-container)
-        * [Test Intel QAT in Docker](#test-intel-qat-in-docker)
+    * [Verify Intel® QAT works in a container](#verify-intel-qat-works-in-a-container)
+        * [Build OpenSSL Intel® QAT engine container](#build-openssl-intel-qat-engine-container)
+        * [Test Intel® QAT with the ctr tool](#test-intel-qat-with-the-ctr-tool)
+        * [Test Intel® QAT in Kubernetes](#test-intel-qat-in-kubernetes)
         * [Troubleshooting](#troubleshooting)
     * [Optional Scripts](#optional-scripts)
-        * [Verify Intel QAT card counters are incremented](#verify-intel-qat-card-counters-are-incremented)
+        * [Verify Intel® QAT card counters are incremented](#verify-intel-qat-card-counters-are-incremented)
 
 # Introduction
 
-Intel QuickAssist Technology (Intel QAT) provides hardware acceleration 
+Intel® QuickAssist Technology (QAT) provides hardware acceleration 
 for security (cryptography) and compression. These instructions cover the 
-steps for [Clear Linux](https://clearlinux.org) but can be adapted to any 
-Linux distribution. Your distribution may already have the Intel QAT 
-drivers, but it is likely they do not contain the necessary user space 
-components. These instructions guide the user on how to download the kernel 
-sources, compile kernel driver modules against those sources, and load them 
-onto the host as well as preparing a specially built Kata Containers kernel 
-and custom Kata Containers rootfs.  
+steps for [Ubuntu 20.04](https://ubuntu.com/download/desktop/thank-you?version=20.04.1&architecture=amd64) 
```

Comment:
version 20.04.1 not found, maybe we shouldn't be quite specific with the version, what about using this link https://ubuntu.com/download/desktop ?